recipes/susiex/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '1.1.0' %}
+
+package:
+  name: susiex
+  version: {{ version }}
+
+source:
+  url: https://github.com/getian107/SuSiEx/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b67dca665e05b1e2a715033c29ad1485b8b3408edcf758d6f5f31a3bd202662c
+  patches:
+    - portable_makefile.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('susiex', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+
+  host:
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+
+  run:
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```

Comment:
These shouldn't be needed - the OpenMP packages have "strong run-exports", which means that listing them in `build` will make sure the correct metadata is added to `host` and `run`.